---
File: Paper_Pool.md
Project: AdvancedBioinfo
Author: Hyunbin Kim (khb7840@gmail.com)
Last Modified: 2021-08-31 22:38:22
---

## Annotation
1. Prokka [paper](https://academic.oup.com/bioinformatics/article/30/14/2068/2390517), [github](https://github.com/tseemann/prokka)
2. Augustus [](https://academic.oup.com/bioinformatics/article/27/6/757/234821), [github](https://github.com/Gaius-Augustus/Augustus)
3. Maker [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-491), [homepage](https://www.yandell-lab.org/software/maker.html)
4. Prodigal [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-119), [github](https://github.com/hyattpd/Prodigal)

## Genome comparison
1. mummer1-4 [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005944), [github](https://github.com/mummer4/mummer)
2. mash [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x), [github](https://github.com/marbl/mash)

## Multiple sequence alignment (MSA)
1. clustalomega [paper](https://www.embopress.org/doi/full/10.1038/msb.2011.75), [github](https://github.com/GSLBiotech/clustal-omega), [homepage](http://www.clustal.org/omega/)
2. mafft [paper](https://academic.oup.com/mbe/article/30/4/772/1073398), [github](https://github.com/GSLBiotech/mafft), [homepage](https://mafft.cbrc.jp/alignment/software/)
3. muscle [paper1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113), [paper2](https://academic.oup.com/nar/article/32/5/1792/2380623), [github](https://github.com/rcedgar/muscle), [homepage](https://www.drive5.com/muscle/)

## Homology search
1. mmseqs2 [paper](https://www.nature.com/articles/nbt.3988), [github](https://github.com/soedinglab/MMseqs2)
2. Diamond1+2 [paper1](https://www.nature.com/articles/s41592-021-01101-x), [paper2](https://www.nature.com/articles/nmeth.3176), [github](https://github.com/bbuchfink/diamond)
3. HHblits [paper](https://www.nature.com/articles/nmeth.1818), [github](https://github.com/soedinglab/hh-suite)

## Transcript
1. stringtie1+2 [paper1](https://www.nature.com/articles/nbt.3122), [paper2](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1910-1), [github](https://github.com/gpertea/stringtie), [homepage](http://ccb.jhu.edu/software/stringtie/)
2. kallisto [paper](https://www.nature.com/articles/nbt.3519), [github](https://github.com/pachterlab/kallisto), [homepage](https://pachterlab.github.io/kallisto/)
3. salmon [paper](https://www.nature.com/articles/nmeth.4197), [github](https://github.com/COMBINE-lab/salmon), [homepage](https://combine-lab.github.io/salmon/)

## Metagenomics
1. metaphlan2 [paper](), [github]()
2. kraken-uniq [paper](), [github]()
3. kraken2 [paper](), [github]()
4. HUMAnN2 [paper](), [github]()

## Phylogenetics
1. raxml [paper](), [github]()
2. fasttree [paper](), [github]()
3. iqtree [paper](), [github]()

## RNA structure prediction
1. viennarna [paper](), [github]()
2. Geometric deep learning of RNA structure [paper](), [github]()

## Protein structure
1. AlphaFold2 [paper](), [github]()
2. RoseTTAFold [paper](), [github]()
3. gromacs [paper](), [github]()

## Workflow and package
1. nextflow [paper](), [github]()
2. snakemake [paper](), [github]()
3. bioconda [paper](), [github]()

## Assembler
1. spades [paper](), [github]()
2. trinity [paper](), [github]()
3. flye [paper](), [github]()
4. hifiasm [paper](), [github]()

## Read aligner
1. hisat1+2 [paper](), [github]()
2. star [paper](), [github]()
3. minimap2 [paper](), [github]()
4. vsearch [paper](), [github]()

## Language models
1. ProtTrans [paper](), [github]()
2. ESM [paper](), [github]()

## Var caller
1. freebayes [paper](), [github]()
2. pilon [paper](), [github]()


- Conterminator [paper](), [github]()
- VG [paper](), [github]()