---
File: README.md
Lecture: Advanced Bioinformatics 1
Professor: Martin Steinegger (martin.steinegger@snu.ac.kr)
TA: Hyunbin Kim (khb7840@snu.ac.kr)
Last Modified: 2021-09-01 13:14:16
---

# Advanced Bioinformatics 1
# The paper pool for Advanced Bioinformatics 2021

## Annotation
1. Prokka [paper](https://academic.oup.com/bioinformatics/article/30/14/2068/2390517), [github](https://github.com/tseemann/prokka)
2. Augustus [paper](https://academic.oup.com/bioinformatics/article/27/6/757/234821), [github](https://github.com/Gaius-Augustus/Augustus)
3. Maker [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-491), [homepage](https://www.yandell-lab.org/software/maker.html)
4. Prodigal [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-119), [github](https://github.com/hyattpd/Prodigal)

## Genome comparison
1. mummer1-4 [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005944), [github](https://github.com/mummer4/mummer)
2. mash [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x), [github](https://github.com/marbl/mash)

## Multiple sequence alignment (MSA)
1. clustalomega [paper](https://www.embopress.org/doi/full/10.1038/msb.2011.75), [github](https://github.com/GSLBiotech/clustal-omega), [homepage](http://www.clustal.org/omega/)
2. mafft [paper](https://academic.oup.com/mbe/article/30/4/772/1073398), [github](https://github.com/GSLBiotech/mafft), [homepage](https://mafft.cbrc.jp/alignment/software/)
3. muscle [paper1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113), [paper2](https://academic.oup.com/nar/article/32/5/1792/2380623), [github](https://github.com/rcedgar/muscle), [homepage](https://www.drive5.com/muscle/)

## Homology search
1. mmseqs2 [paper](https://www.nature.com/articles/nbt.3988), [github](https://github.com/soedinglab/MMseqs2)
2. Diamond1+2 [paper1](https://www.nature.com/articles/s41592-021-01101-x), [paper2](https://www.nature.com/articles/nmeth.3176), [github](https://github.com/bbuchfink/diamond)
3. HHblits [paper](https://www.nature.com/articles/nmeth.1818), [github](https://github.com/soedinglab/hh-suite)

## Transcript
1. stringtie1+2 [paper1](https://www.nature.com/articles/nbt.3122), [paper2](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1910-1), [github](https://github.com/gpertea/stringtie), [homepage](http://ccb.jhu.edu/software/stringtie/)
2. kallisto [paper](https://www.nature.com/articles/nbt.3519), [github](https://github.com/pachterlab/kallisto), [homepage](https://pachterlab.github.io/kallisto/)
3. salmon [paper](https://www.nature.com/articles/nmeth.4197), [github](https://github.com/COMBINE-lab/salmon), [homepage](https://combine-lab.github.io/salmon/)

## Metagenomics
1. metaphlan2 [metaphlan1_paper](https://www.nature.com/articles/nmeth.2066), [metaphlan2_paper](https://www.nature.com/articles/nmeth.3589), [github](https://github.com/biobakery/MetaPhlAn2), [homepage](https://huttenhower.sph.harvard.edu/metaphlan2/)
2. kraken-uniq [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1568-0), [github](https://github.com/fbreitwieser/krakenuniq)
3. kraken2 [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0), [github](https://github.com/DerrickWood/kraken2)
4. HUMAnN2 [paper](https://www.nature.com/articles/s41592-018-0176-y), [github](https://github.com/biobakery/humann), [homepage](https://huttenhower.sph.harvard.edu/humann2/)

## Phylogenetics
1. raxml [paper](https://academic.oup.com/bioinformatics/article/30/9/1312/238053), [github](https://github.com/stamatak/standard-RAxML), [homepage](https://cme.h-its.org/exelixis//web/software/raxml/index.html)
2. fasttree [paper1](https://academic.oup.com/mbe/article/26/7/1641/1128976),[paper2](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009490), [homepage](http://www.microbesonline.org/fasttree/)
3. iqtree [paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [github](https://github.com/Cibiv/IQ-TREE), [homepage](http://www.iqtree.org/)

## RNA structure prediction
1. viennarna [paper](https://almob.biomedcentral.com/articles/10.1186/1748-7188-6-26), [github](https://github.com/ViennaRNA/ViennaRNA), [homepage](https://www.tbi.univie.ac.at/RNA/)
2. Geometric deep learning of RNA structure [paper](https://www.science.org/doi/abs/10.1126/science.abe5650), [zenodo1](https://zenodo.org/record/5088971), [zenodo2](https://zenodo.org/record/5090151), [zenodo3](https://zenodo.org/record/5090157)

## Protein structure
1. AlphaFold2 [paper](https://www.nature.com/articles/s41586-021-03819-2), [github](https://github.com/deepmind/alphafold)
2. RoseTTAFold [paper](https://www.science.org/doi/abs/10.1126/science.abj8754), [github](https://github.com/RosettaCommons/RoseTTAFold)
3. gromacs [paper](https://www.sciencedirect.com/science/article/pii/S2352711015000059?via%3Dihub), [github](https://github.com/gromacs/gromacs), [homepage](https://www.gromacs.org/)

## Workflow and package
1. nextflow [paper](https://www.nature.com/articles/nbt.3820), [github](https://github.com/nextflow-io/nextflow), [homepage](https://www.nextflow.io/)
2. snakemake [paper](https://academic.oup.com/bioinformatics/article/28/19/2520/290322), [github](https://github.com/snakemake/snakemake), [documentation](https://snakemake.readthedocs.io/en/stable/)
3. bioconda [paper](https://www.nature.com/articles/s41592-018-0046-7), [github](https://github.com/bioconda)
4. bioconductor [paper1](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2004-5-10-r80), [paper2](https://www.nature.com/articles/nmeth.3252), [github](https://github.com/Bioconductor), [homepage](https://www.bioconductor.org/)

## Assembler
1. spades [paper](https://www.liebertpub.com/doi/full/10.1089/cmb.2012.0021), [github](https://github.com/ablab/spades), [homepage](https://cab.spbu.ru/software/spades/)
2. trinity [paper](https://www.nature.com/articles/nbt.1883), [github](https://github.com/trinityrnaseq/trinityrnaseq)
3. flye [paper](https://www.nature.com/articles/s41587-019-0072-8), [github](https://github.com/fenderglass/Flye)
4. hifiasm [paper](https://www.nature.com/articles/s41592-020-01056-5), [github](https://github.com/chhylp123/hifiasm)

## Read aligner
1. hisat1+2 [paper](https://www.nature.com/articles/s41587-019-0201-4), [github](https://github.com/DaehwanKimLab/hisat2), [homepage](http://daehwankimlab.github.io/hisat2/)
2. star [paper](https://academic.oup.com/bioinformatics/article/29/1/15/272537), [github](https://github.com/alexdobin/STAR)
3. minimap2 [paper](https://academic.oup.com/bioinformatics/article/34/18/3094/4994778), [github](https://github.com/lh3/minimap2)
4. vsearch [paper](https://peerj.com/articles/2584/), [github](https://github.com/torognes/vsearch)

## Language models
1. ProtTrans [paper](https://ieeexplore.ieee.org/document/9477085), [github](https://github.com/agemagician/ProtTrans)
2. ESM [paper](https://www.biorxiv.org/content/10.1101/622803v4), [github](https://github.com/facebookresearch/esm)
3. DNABERT [paper](https://doi.org/10.1093/bioinformatics/btab083), [github](https://github.com/jerryji1993/DNABERT)

## Variant caller
1. freebayes [paper](https://arxiv.org/abs/1207.3907), [github](https://github.com/freebayes/freebayes)
2. pilon [paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112963), [github](https://github.com/broadinstitute/pilon)
3. gatk [paper1](https://www.nature.com/articles/ng.806), [paper2](https://www.biorxiv.org/content/10.1101/201178v3.full.pdf), [github](https://github.com/broadinstitute/gatk), [homepage](https://gatk.broadinstitute.org/hc/en-us)

## Other topics
1. Conterminator [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1), [github](https://github.com/martin-steinegger/conterminator)
2. VG [paper](https://www.nature.com/articles/nbt.4227), [github](https://github.com/vgteam/vg)
